src/main/kotlin/sidechain/iroha/IrohaChainListener.kt

```diff
@@ -34,10 +40,8 @@ class IrohaChainListener : ChainListener<IrohaBlockStub> {
             val scheduler = Schedulers.from(Executors.newSingleThreadExecutor())
 
             stub.fetchCommits(query).toObservable().map {
-                logger.info { "New block" }
-                val file = File("resources/genesis.bin")
-                val bs = file.readBytes()
-                IrohaBlockStub.fromProto(bs)
+                //TODO x3medima17 27.06.2018, implement all commands fromProto
+                IrohaBlockStub.fromProto(it.blockResponse.block.toByteArray())
```

Comment:
I suggest to return `it.blockResponse.block` here and parse and emit `ChainEvent.IrohaEvent` in `IrohaHandler`.